gnomad_qc/v4/assessment/calculate_per_sample_stats.py

```diff
@@ -513,7 +583,17 @@ def main(args):
         "--slack-channel", help="Slack channel to post results and notifications to."
     )
     parser.add_argument(
-        "--test",
+        "--test-n-partitions",
+        type=int,
+        help="Number of partitions to use for testing.",
+    )
+    parser.add_argument(
+        "--test-difficult-partitions",
+        help="Whether to test on a set of 4 difficult exome partitions.",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--test-dataset",
         help="Test on a small number of variants",
         action="store_true",
```

Comment:
I think we have to specify this: VDS will be using VDS of a small number of samples, and release_ht only chr22.